mobx/lib/src/api/annotations.dart

```diff
@@ -1,3 +1,10 @@
+class MakeStore {
+  const MakeStore._();
+}
+
+/// Declares a class as a store.
+const MakeStore store = MakeStore._();
```

Comment:
You may want to publish an alpha version package of `mobx`, which the `mobx_codegen` can depend on. The alpha version are ignored by pub as they are not included in the version check

Reply:
We also need a bunch of tests / examples

Reply:
Sounds like a plan. I'm going to split this into two PRs. The first to update mobx. Once we have that in, would you mind reporting back once you've published? I'll update this PR's pubspec to match.

Reply:
I can't publish the alpha version myself, can I?